bodhi/tests/messages/schemas/test_update.py

```diff
@@ -36,7 +36,7 @@
 from bodhi.tests.messages.utils import check_message
 
 
-class UpdateMessageTests(unittest.TestCase):
+class UpdateMessageTests:
```

Comment:
In order to be recognized as a test, the class name has to start with `Test`. Would you rename it to `TestUpdateMessage`? Thanks!

Reply:
Yes sure I'll